main.py

```diff
@@ -21,24 +26,46 @@
 
 CDC_CRITERIA_1_GOOGLE_WORKBOOK_KEY = "1p4Z6zTa6O0ss5B5rgoWotAIwsBqqEqwdcM3Yel-mm5g"
 CDC_CRITERIA_2_GOOGLE_WORKBOOK_KEY = "1xdePOZkXXv49_15YTloLr7D72eQhY9R-ZEhoMr-4UY0"
+CDC_CRITERIA_3_GOOGLE_WORKBOOK_KEY = "1-BSd5eFbNsypygMkhuGX1OWoUsF2u4chpsu6aC4cgVo"
 
 # Note: if you'd like to run the full pipeline, you'll need to generate a service account keyfile for an account
 # that has been given write access to the Google Sheet.
-PATH_TO_SERVICE_ACCOUNT_KEY = "service-account-key.json"
 
 
 def extract_transform_and_load_covid_data():
-    df = extract_covidtracking_historical_data()
-
-    transformed_df = transform_covidtracking_data(df=df)
-
     client, credentials = get_sheets_client(
         credential_file_path=os.path.abspath(PATH_TO_SERVICE_ACCOUNT_KEY)
     )
 
+    cdc_current_df = extract_current_cdc_data()
+    cdc_historical_df = extract_historical_cdc_data(credentials)
```

Comment:
The flow here is that we pull the new data using Brice's scraper, we pull our "db" data from a google sheet, we mash the two to get all the historical data + the current data on the CDC site, then transform and upload.

Reply:
@lucasmbrown ^

Reply:
Each time, the new CDC data is added to our historical data set